Sources/RInAppMessaging/Constants.swift

```diff
@@ -45,6 +46,12 @@ internal enum Constants {
 
     enum Versions {
         static let sdkVersion = "8.1.0-snapshot"
+        static var rmcSdkVersion: String? {
+            if let rmcBundle = Bundle(identifier: "RMC-RMC-resources") {
```

Comment:
what is `RMC-RMC-resources`?

Reply:
@anish-munirathinam That's the identifier of the resource bundle that will be defined in RMC SDK's `Package.swift`